lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -66,11 +66,19 @@ function dosomething_reportback_build_reportbacks_query($params = array()) {
   }
 
   if (isset($params['runs'])) {
-    if (is_array($params['runs'])) {
```

Comment:
@aaronschachter I tried to change line 68 to add `&& is_numeric($params['runs'])` when testing with `runs=z` but since it wasn't numeric, it skipped this entire conditional and still returned results when we want it to return an empty data object. 

In order to weed out the param values that aren't numeric for `runs`, I think it makes most sense to iterate through them all and put the valid ones in an array and then run the conditional clause against those that we know are definitely numbers? 

Refresher - on production and on thor, when `runs=z`, we get a signup object with the `run_id` of `0` returned as well - seems like the version of php matches the numbers to pass through to `0` for some reason.